index.html

```diff
@@ -6,40 +6,56 @@
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
-
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  
 </head>
 
+<header>
+    <div class="nav-container">
+    <img src="img/lambda-black.png">
+    <nav>
+        <ul>
+            <li><a href="#">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Products</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
+    </div>
+</header>
+
 <body>
     <div class="container">
-   
-        <section class="top-content">
-            <div class="text-container">
-                <h2>The Future</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-            <div class="text-container">
-                <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-        </section>
+        <img src="img/jumbo.jpg">
+
+    <section class="top-content">
+        <div class="text-container">
+            <h2>The Future</h2>
+            <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+        </div>
+        <div class="text-container">
+            <h2>The Past</h2>
+            <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+        </div>
+    </section>
         
         <section class="middle-content">
         
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" style="background:teal;">Box 1</div>
```

Comment:
Great job using inline styling to set the colors of the box, however, mixing inline styles with external styling libraries can get messy. Instead of storing your styles in one place, now you have 2.

One of the main principles of programming is writing code that is reusable. In this case, for each of the boxes you want all the boxes the same size, but you want them different colors. One of the ways you could achieve this is by keeping the "box" class that assigns that specific element its shape and add an additional class to each box to specify the color of the box. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```

```
.box{
   /* styles to make a box*/
}

.box1 {
    /* color box gree*/
}

.box2{
    /* color box purple*/
}
```